README.md

```diff
@@ -3,6 +3,8 @@
 This repository contains the current, and historical, JSON Schema
 specifications.
 
+It should be noted that the master branch of this repo does not constitute a currently released version.
+For draft 3 or 4, please see <https://tools.ietf.org/html/draft-zyp-json-schema-03> and <https://tools.ietf.org/html/draft-zyp-json-schema-04> respectivly
```

Comment:
Maybe we could do both? I mean, personally, if I'm looking for the latest version, I'd rather be directed to the draft document on ietf, so I think this is fine. Plus I'm not sure we have tags for 3 and 4 in this repo.
(Probably should add draft 5, right?)